src/brain.coffee

```diff
@@ -804,6 +807,9 @@ class Brain
       else
         random = text.split(" ")
 
+      # Delete any empty randoms.
+      # random = random.filter (val)-> val != ''
```

Comment:
Although added, I commented this out specifically in the situation that a writer might want to return an empty value.